src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +117,23 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  /* Return value here is a tuple: (workflow name, call name) */
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+    callFqn.split("\\.").toSeq match {
```

Comment:
Can you add a FIXME here (just to make it obvious) that in a world where FQN isn't a type alias that this sort of thing should be pulled into there?